requirements.txt

```diff
@@ -4,5 +4,5 @@ itsdangerous==1.1.0
 Jinja2==2.11.3
 MarkupSafe==1.1.1
 Werkzeug==1.0.1
-numpy==1.23.1
+numpy==1.21.6
```

Comment:
Why do we use an older version of NUMPY?

Reply:
The suggestion to change `numpy==1.21.6` to just `numpy` worked. Now all the docker images download the most recent version of numpy for the respective python version.